addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -283,6 +285,53 @@ var Wysiwyg = Widget.extend({
             mediaDialog.on('cancel', this, resolve);
         });
     },
+    changeSelectedTextColor(color, closeColorPicker = false) {
+        if(color === "") {
+            this.editor.execCommand('uncolorText');
+            $(window).trigger('click');
+            return;
+        }
+
+        if(this.colorPicker.colorNames.indexOf(color) !== -1) {
+           color = "var(--" + color + ")";
+        }
+        this.editor.execCommand('colorText', {color: color});
+
+        if(closeColorPicker) {
+            $(window).trigger('click');
+        }
+    },
+    toggleColorPicker() {
+        return new Promise((resolve)=> {
+            if(this.colorPicker) {
+                $(window).trigger('click');
+                return;
+            }
+            const self = this;
```

Comment:
No need to use self, you are using the fat arrow function.

Reply:
I still need the self in the `closeColorPicker` function